research-hub-web/src/app/services/notification.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { NotificationService } from './notification.service';
+
+describe('NotificationService', () => {
+  let service: NotificationService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(NotificationService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
as discussed it would be good to add some test cases, but we can capture that they're missing when I do the test review. The notification component also doesn't have any specific tests, so I think it's something we can come back to address later.